<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BestWorkdayWidget</class>
 <widget class="QWidget" name="BestWorkdayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>.Helvetica Neue Desk UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Best Workday</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBestWorkdayName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans Unicode</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>workdayNamePlaceholder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBestWorkdayMsg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>.Helvetica Neue Desk UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Average comparison message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="sprint_timer::ui::qt_gui::BarChart" name="workdayBarChart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sprint_timer::ui::qt_gui::BarChart</class>
   <extends>QWidget</extends>
   <header>src/widgets/BarChart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
